pipeline/metadata/flatten_satellite.py

```diff
@@ -287,6 +287,8 @@ def _process_satellite_v1(self, date: str, responses_entry: ResponsesEntry,
             responses_entry['query'],
         'is_control':
             False,  # v1 doesn't have domain controls
+        'controls_failed':
+            False,  # v1 doesn't have domain controls
```

Comment:
If we don't have a control, shouldn't we assume it failed? Its absence sounds equivalent to a failure.

Reply:
We filter out measurements with failed controls. So if we made this True we would filter out 100% of the sat v1 measurements.

The field is not "did a control succeed", it's "did any related controls fail"

so running 0 controls -> 0 controls failed -> False

Reply:
What control are we talking about? The control resolver or the control domain? We need better names... Perhaps some renaming early would indeed help. It doesn't have to be the final names. We should be able to rename easily.

There's also a data modeling issue here. Why are we merging control and non control data?